content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+<h4>MINIMUM (32-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Pentium II; AMD Athlon</li>
+<li><strong>Memory:</strong> 256MB</li>
+<li><strong>Monitor:</strong> 800x600</li>
+<li><strong>Storage:</strong> 3GB</li>
+</ul>
+</td><td>
+<h4>RECOMMENDED (64-bit)</h4>
+<ul>
+<li><strong>Processor</strong>: Intel Core i3; AMD Phenom II</li>
+<li><strong>Memory:</strong> 2GB</li>
+<li><strong>Monitor:</strong> 1366x768</li>
+<li><strong>Storage:</strong> 16GB</li>
+</ul>
+</td></tr></table>
+
+SSE2 support is required to use the WebPositive web browser. On machines where this is not available, it is recommended to install the NetSurf browser instead.
+
+Haiku takes advantage of modern hardware and will run faster on newer machines. 
+
+## Upgrading from Beta 1
+
+There is an upgrade path available for users that are currently using Haiku R1 Beta 1. If you want to do this upgrade, you will have to use the Terminal to issue some commmands.
+
+*Warning: only upgrading from Beta 1 is supported. It may be possible to upgrade from other intermediate development images, but this is untested and you may run into unknown problems.*
+
+### Step 1: Getting the Latest Updates
+
+You need to make sure that you have all the package updates that are available for your current release. You can do this by running the following command in the terminal:
+
+```shell script
+pkgman update
+```
+
+### Step 2: Switch to the new repositories
+After this we will switch the package manager to point to the new repositories for beta 2. This will allow the package manager to find the new packages. Run the following commands in the Terminal.
+
+```shell script
+pkgman drop Haiku; pkgman drop HaikuPorts;
+pkgman add-repo https://eu.hpkg.haiku-os.org/haiku/r1beta2/$(getarch)/current
+pkgman add-repo https://eu.hpkg.haiku-os.org/haikuports/r1beta2/$(getarch)/current
+```
+
+### Step 3: Installing the new packages and reboot
+
+Now it is time to download and install the packages. You can do this by issuing the following command in the Terminal:
+
+```shell script
+pkgman full-sync
+```
+
+After that step has been successful, you should immediately reboot the system. You can do this from the Deskbar, or by running ```shutdown -r``` in the terminal.
+
+## Beta 2 Highlights
+
+This release contains 20 months worth of improvements. Below you find some highlights.
+
+### New Input Preferences
+
+The Mouse, Keyboard, Touchpad and Joystick preferences have been consolidated into one Input Preferences.
+
+<img src="/files/get-haiku/input_preferences.png"/><br>
+<small><i>The consolidated Input Preferences, including support for mice with 5 buttons.</i></small>
+
+Additionally, there is now support for mice with more than three buttons.
+
+### DeskBar improvements
+
+The Deskbar has had various improvements. It now has a mini-mode, where the icons on the shelf expand leftwards. You can activate it by dragging the shelf space by the right handle to the top of the screen.
```

Comment:
It already had a mini mode, the new one just looks different (it puts everything on one line, before there would be one line with deskbar menu + teams menu, and the shelf below)